frontend/.env.example

```diff
@@ -0,0 +1,3 @@
+VITE_API_ORIGIN_URL=http://localhost:3001/api/v1
+VITE_SENTRY_DSN='your sentry dsn'
+SENTRY_ENABLE=false
```

Comment:
`SENTRY_ENABLE` also needs to have the `VITE_` prefix. W/o it, the variable won't be recognized
https://vitejs.dev/guide/env-and-mode.html#env-files

Reply:
We also miss the `VITE_SENTRY_DSN`